apps/new/widget/page/projects/Editor.jsx

```diff
@@ -610,7 +610,7 @@ const SecondScreen = () => {
         <div className="form-group mb-3">
           <label className="pb-2">Avatar</label>
           <Widget
-            src="${alias_old}/widget/components.ImageUploader"
+            src="${alias_old}/widget/components.UploadField"
```

Comment:
Can you update UploadField so that it uses Files component to enable drag and drop support?
https://docs.near.org/build/near-components/anatomy/builtin-components#files

Reply:
Sure will implement it